vibration/api-is-present.html

```diff
@@ -19,7 +19,7 @@ <h1>Description</h1>
     <script src='/resources/testharnessreport.js'></script>
     <script>
       test(function () {
-          assert_true(undefined !== navigator.vibrate, "navigator.vibrate exists");
+          assert_not_equals(undefined, navigator.vibrate, "navigator.vibrate exists");
```

Comment:
Optional nit (at the original test :D)

```suggestion
          assert_not_equals(navigator.vibrate, undefined, "navigator.vibrate exists");
```